### PR TITLE
Read replicasets from ui-api-layer

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- read replica sets from ui-api-layer

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/2725

### CODE REVIEW

**Comment by kwiatekus** [comment]
Kyma counter part for those changes are available in the following branch:
[kwiatekus:replicasets-graphql-e2e](https://github.com/kwiatekus/kyma/tree/replicasets-graphql-e2e)

File: core/src/app/content/environments/operation/abstract-graphql-element-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          this.graphQLClientService
+        );
+      });
+  }
+
+  public ngOnDestroy() {
+    this.currentEnvironmentSubscription.unsubscribe();
+  }
+
+  protected getReadAllQuery() {
```
Comment (dariadomagala) [inline core/src/app/content/environments/operation/abstract-graphql-element-list.component.ts]: Get read all query? I have no idea what this function should do 😄 Could you please change the name or explain it somehow?

File: core/src/app/content/environments/operation/replica-sets/replica-sets-header-renderer/replica-sets-header-renderer.component.html [hunk 2]
```diff
@@ -1,9 +1,8 @@
 <div class="row sf-list__head">
-  <div class="col-2 col-lg-2">Name</div>
+  <div class="col-3 col-lg-3">Name</div>
   <div class="col-1">Pods</div>
   <div class="col-1">Age</div>
-  <div class="col-3">Images</div>
+  <div class="col-4">Images</div>
   <div class="col-2">Labels</div>
-  <div class="col-2">Status</div>
```
Comment (dariadomagala) [inline core/src/app/content/environments/operation/replica-sets/replica-sets-header-renderer/replica-sets-header-renderer.component.html]: what is the reason we don't have status anymore?

File: core/src/app/content/environments/operation/abstract-graphql-element-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+  getResourceJSONQuery() {
+    return `query ${this.resourceKind}($name: String! $namespace: String!) {
+      replicaSet(name: $name, namespace: $namespace) {
+        json
+      }
+    }`;
+  }
+
+  sendDeleteRequest(entry) {
```
Comment (dariadomagala) [inline core/src/app/content/environments/operation/abstract-graphql-element-list.component.ts]: I was not able to delete my RS.
screen shot 2019-02-25 at 10 54 49

Comment (dariadomagala) [reply core/src/app/content/environments/operation/abstract-graphql-element-list.component.ts]: Same for editing. Mutations don't work for me.

**Pull request merged**